app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,104 @@
+<resources>
+    <string name="app_name">Food Expiration Dates</string>
+    <string name="app_name_short">Food Expirations</string>
+    <string name="n_days_ago">%1$d gün önce</string>
+    <string name="yesterday">Dün</string>
+    <string name="today">Bugün</string>
+    <string name="tomorrow">Yarın</string>
+    <string name="in_n_days">%1$d gün içinde</string>
+    <string name="add_item">Öğe ekle</string>
+    <string name="edit_item">Öğeyi düzenle</string>
+    <string name="food_name">Yemek adı</string>
+    <string name="expiration_date">Son kullanma tarihi</string>
+    <string name="please_select_a_date">Lütfen bir tarih seçin</string>
+    <string name="please_enter_a_food_name">Lütfen bir yemek adı girin</string>
+    <string name="insert">Ekle</string>
+    <string name="update">Güncelle</string>
+    <string name="cancel">İptal</string>
+    <string name="share">Paylaş</string>
+    <string name="leave_a_star_on_github">GitHub\'da bir yıldız bırak</string>
+    <string name="write_a_review">Değerlendirme yaz</string>
+    <string name="source_code">Kaynak kodu</string>
+    <string name="features">Özellikler</string>
+    <string name="feature_list">
+        - Artan zaman sırasına göre yemek son kullanma tarihlerini içeren bir liste görüntüleyin\n
+        - Öğe ekleyin, düzenleyin ve silin\n
+        - Açılış tarihi\n
+        - Bildirimler
+    </string>
+    <string name="support_this_project">Bu projeyi destekle</string>
+    <string name="report_a_bug">Hata bildir</string>
+    <string name="version_x">Sürüm %1$s</string>
+    <string name="app_description">%1$s, Lorenzo Vainigli ve diğer katkıda bulunanlar tarafından geliştirilen açık kaynaklı bir projedir.\nBu uygulamanın kodu MIT lisansı altında yayınlanmıştır ve GitHub'da mevcuttur.</string>
```

Comment:
Can you escape the character `'`  in `GitHub'da` as follows? Otherwise the build fails.
```suggestion
    <string name="app_description">%1$s, Lorenzo Vainigli ve diğer katkıda bulunanlar tarafından geliştirilen açık kaynaklı bir projedir.\nBu uygulamanın kodu MIT lisansı altında yayınlanmıştır ve GitHub\'da mevcuttur.</string>
```

Reply:
fixed